test/python/tests/cli_controller/docker-compose.yml

```diff
@@ -0,0 +1,92 @@
+#
+# {SRCDIR} should point to your working tree which should be your current pwd
+#
+
+version: '3'
+services:
+  ms1:
```

Comment:
Does the naming of containers follow a certain format? here it starts with ms1, but in some files it starts with ms0

Reply:
See comments above. Yes this test probably should use "ms0" as its first container. However, I did not write this test, and decided to leave it as is - at least for now.